cloud/aws/metrics/adot-config.yaml

```diff
@@ -0,0 +1,54 @@
+receivers:
```

Comment:
If possible would be good to add comment stating what this file is and reference to some docs about this config format.

Reply:
done